Src/FluentAssertions/Equivalency/Steps/DictionaryInterfaceInfo.cs

```diff
@@ -37,10 +37,8 @@ private DictionaryInterfaceInfo(Type key, Type value)
     /// <remarks>>
     /// The <paramref name="role"/> is used to describe the <paramref name="target"/> in failure messages.
     /// </remarks>
-    public static bool TryGetFrom(Type target, string role, out DictionaryInterfaceInfo result)
+    public static DictionaryInterfaceInfo FindFrom(Type target, string role)
```

Comment:
🤔 I actually prefer the original `TryGetFrom`, but don't hold that preference strongly.